pkg/skaffold/schema/defaults/defaults.go

```diff
@@ -96,7 +96,10 @@ func Set(c *latest.SkaffoldConfig) error {
 		return err
 	}
 
-	for _, pf := range c.PortForward {
+	for i, pf := range c.PortForward {
+		if pf == nil {
+			return fmt.Errorf("portForward[%d] of config name with '%s' is empty, Please check if it has valid values", i, c.Metadata.Name)
```

Comment:
```suggestion
			return fmt.Errorf("portForward[%d] of config with name '%s' is empty, Please check if it has valid values", i, c.Metadata.Name)
```
I think the position of this word should be flipped

Reply:
Right, This should be 😅